### DISCUSSION TITLE
Issue with createReceiptShipment operation

### DISCUSSION THREAD

**Discussion by rkistaps** [etsy/open-api#726] [Q&A] [answered]
Hello.

We are seeing some weird behavior for the createReceiptShipment operation for some of our customers. In some cases, when we try to submit tracking info, we get back 404 with an error message that the receipt is not associated with the shop. But when using the "getShop_Receipt2" method(from the v2 API), we see the receipt in the store. What could be the reason for the receipt not being found when trying to add tracking info?

For example:

We make POST /v3/application/shops/34543415/receipts/2721069404/tracking with body:

```
{
    "carrier_name": "sf-express",
    "tracking_code": "SF6043506624341",
    "receipt_id": 2721069404
}
```

And get the response with a status code 404 with the response:
```
{
    "headers": {
        "X-Etsy-Request-Uuid": "EuRrMkw_9APUJaUfxcCuPevL6m5f",
        "X-Limit-Per-Second": "20",
        "X-Remaining-This-Second": "19",
        "X-Limit-Per-Day": "50000",
        "X-Remaining-Today": "49639"
    },
    "data": {
        "error": "Could not find a Shop Receipt with receipt_id = 2721069404 associated with shop_id = 34543415."
    }
}
```

Any insight is greatly appreciated.

**Comment by etsyachristensen** [comment, marked as answer]
That receipt is a cancelled order. OpenAPI V3 does not return cancelled orders nor will it allow you to set a shipment for one.
